### PR TITLE
KOGITO-1605: Amend issues in GitHub documentation

### PR DESCRIPTION
**Pull request by Sgitario**
- Wrong kogito-imagestream.yaml link
- Infinispan link not working
- Wrong sentence in Running the Kogito Operator locally
- Add requirements for testing e2e

JIRA Link: https://issues.redhat.com/browse/KOGITO-1605

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1152,8 +1159,6 @@ $ make clean
 $ DEBUG=true operator-sdk run --local --namespace=<namespace>
 ```
 
-Before submitting a [pull request](https://help.github.com/en/articles/about-pull-requests) to the Kogito Operator repository, review the instructions for [Contributing to the Kogito Operator](docs/CONTRIBUTING.MD).
```
Comment (sutaakar) [inline README.md]: Why is this removed?

File: README.md [hunk 2]
```diff
@@ -969,6 +969,13 @@ The information required to connect to Kafka will be automatically set for you b
 
 Before you begin fixing issues or adding new features to the Kogito Operator, see [Contributing to the Kogito Operator](docs/CONTRIBUTING.MD) and [Kogito Operator architecture](docs/ARCHITECTURE.MD).
 
+### Requirements
```
Comment (sutaakar) [inline README.md]: Please add this section to Table of Contents

File: README.md (hunk 1 above)
Comment (Sgitario) [reply README.md]: This section is at the bottom of the article (just a couple of lines later). So, this line didn't match with the topic of the current section.

File: README.md [hunk 3]
```diff
@@ -969,6 +969,13 @@ The information required to connect to Kafka will be automatically set for you b
...
+
+- [Docker](https://www.docker.com/)
+- [Operator Courier](https://github.com/operator-framework/operator-courier) is used to build, validate and push Operator Artifacts
+- [Operator SDK](https://github.com/operator-framework/operator-sdk) V0.16.0 or later
```
Comment (sutaakar) [inline README.md]: AFAIK operator-sdk 0.15.x can be used too.

File: README.md [hunk 4]
```diff
@@ -969,6 +969,13 @@ The information required to connect to Kafka will be automatically set for you b
...
+- [Golint dependency](golang.org/x/lint/golint): go get -u golang.org/x/lint/golint
```
Comment (sutaakar) [inline README.md]: I think we should also mention Go 1.13.x requirement, see chapter Kogito Operator requirements

File: README.md (hunk 2 above)
Comment (sutaakar) [reply README.md]: Maybe would be better to rename to Build requirements, as there is already `Kogito Operator requirements` section too.

File: README.md (hunk 3 above)
Comment (Sgitario) [reply README.md]: True. Fixed.

File: README.md (hunk 4 above)
Comment (Sgitario) [reply README.md]: Added

File: README.md (hunk 2 above)
Comment (Sgitario) [reply README.md]: Done

File: README.md [hunk 5]
```diff
@@ -969,6 +970,14 @@ The information required to connect to Kafka will be automatically set for you b
 
 Before you begin fixing issues or adding new features to the Kogito Operator, see [Contributing to the Kogito Operator](docs/CONTRIBUTING.MD) and [Kogito Operator architecture](docs/ARCHITECTURE.MD).
 
+### Requirements
+
+- [Docker](https://www.docker.com/)
```
Comment (sutaakar) [inline README.md]: AFAIK it should be possible to use also Podman instead of Docker for operator build.

**Comment by sutaakar** [comment]
@Sgitario Can you please select applied steps from the list in https://github.com/kiegroup/kogito-cloud-operator/pull/248#issue-395447057 ?
And also apply missing steps where possible?

File: README.md (hunk 5 above)
Comment (spolti) [reply README.md]: +1 in podman or buildah as as preferreable build tool

File: README.md (hunk 1 above)
Comment (spolti) [reply README.md]: where exactly?

Comment (ricardozanini) [reply README.md]: This paragraph should be in the "Development" session. Please move it there.

Comment (Sgitario) [reply README.md]: What I was trying to say is that this topic is covered in the last section: https://github.com/kiegroup/kogito-cloud-operator#contributing-to-the-kogito-operator
which is saying the same but using other words.
What do you think?

File: README.md (hunk 5 above)
Comment (Sgitario) [reply README.md]: Done. I had only set Docker because it's the one used by default.

File: README.md (hunk 1 above)
Comment (ricardozanini) [reply README.md]: You're right! Thanks for the input.

**Comment by ricardozanini** [comment]
@Sgitario please squash your commits before we merge.

**Comment by Sgitario** [comment]
Squash done.
